Fix:
error: implicit declaration of function 'utime' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
error: implicit declaration of function 'kill' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
https://github.com/kholtman/afio/issues/17
--- afio.c.orig	2018-11-30 08:25:04.000000000 -0600
+++ afio.c	2022-01-26 17:24:28.000000000 -0600
@@ -162,6 +162,8 @@
 
 #include <stdio.h>
 #include <errno.h>
+#include <utime.h>
+#include <signal.h>
 
 #ifdef sun
 #include <sys/types.h>
